runtime/rasdump/TextFileStream.cpp

```diff
@@ -73,7 +74,9 @@ TextFileStream::close(void)
 			j9file_write_text(_FileHandle, _Buffer, _BufferPos);
 		}
 		j9file_sync(_FileHandle);
-		j9file_close(_FileHandle);
+		if (_FileHandle != OMRPORT_TTY_OUT && _FileHandle != OMRPORT_TTY_ERR) {
```

Comment:
Please be consistent in the constant names. Above on line 56, 58, J9PORT is used instead of OMRPORT. I think using OMRPORT is fine. The J9PORT constants were added to avoid having to fix all the code when the port library moved to OMR and got renamed from J9 to OMR.

Reply:
Sorry, I must've copied the variable name wrong... Clumsy mistake

Reply:
Better than checking the value of `_FileHandle`, I think a boolean variable set if the call to `j9file_open` succeeds should be the condition for calling `j9file_close`.